profile/ucs_domain_profile/terraform/ucs_domain_profile.tf

```diff
@@ -1,10 +1,238 @@
+    moid        = var.organization
+  }
+}
+
+resource "intersight_fabric_switch_control_policy" "fabric_switch_control_policy1" {
+  name        = "fabric_switch_control_policy1"
+  description = "fabric switch control policy"
+  mac_aging_settings {
+    mac_aging_option = "Custom"
+    mac_aging_time   = 3000
+    object_type      = "fabric.MacAgingSettings"
+  }
+  vlan_port_optimization_enabled = true
+  organization {
+    object_type = "organization.Organization"
+    moid        = var.organization
+  }
+}
+
+resource "intersight_ntp_policy" "ntp1" {
+  name        = "ntp1"
+  description = "test policy"
+  enabled     = true
+  ntp_servers = [
+    "ntp.esl.cisco.com",
+    "time-a-g.nist.gov",
+    "time-b-g.nist.gov"
+  ]
+  organization {
+    object_type = "organization.Organization"
+    moid        = var.organization
+  }
+}
+
+resource "intersight_networkconfig_policy" "network_config1" {
+  name                     = "network_config1"
+  description              = "demo network configuration policy"
+  enable_dynamic_dns       = false
+  preferred_ipv6dns_server = "::"
+  enable_ipv6              = true
+  enable_ipv6dns_from_dhcp = false
+  preferred_ipv4dns_server = "10.10.10.1"
+  alternate_ipv4dns_server = "10.10.10.1"
+  alternate_ipv6dns_server = "::"
+  dynamic_dns_domain       = ""
+  enable_ipv4dns_from_dhcp = false
+  organization {
+    object_type = "organization.Organization"
+    moid        = var.organization
+  }
+}
+
+# Create a cluster switch profile
+resource "intersight_fabric_switch_cluster_profile" "cluster_switch_profile_1" {
+  name = "cluster_switch_profile_1"
+  organization {
+    object_type = "organization.Organization"
+    moid        = var.organization
+  }
+}
+
+# Create a domain profile for Fabric switch A
+resource "intersight_fabric_switch_profile" "fabric_A_profile" {
+  name                = "Fabric_A_profile"
+  switch_cluster_profile {
+    object_type = "fabric.SwitchClusterProfile"
+    moid        = intersight_fabric_switch_cluster_profile.cluster_switch_profile_1.moid
+  }
+  policy_bucket = [
+    {
+      additional_properties = ""
+      class_id              = "fabric.EthNetworkPolicy"
+      object_type           = "fabric.EthNetworkPolicy"
+      moid                  = intersight_fabric_eth_network_policy.fabric_eth_network_policy1.moid
+      selector              = ""
+    },
+    {
+      additional_properties = ""
+      class_id              = "fabric.FcNetworkPolicy"
+      object_type           = "fabric.FcNetworkPolicy"
+      moid                  = intersight_fabric_fc_network_policy.fabric_fc_network_policy1.moid
+      selector              = ""
+    },
+    {
+      additional_properties = ""
+      class_id              = "fabric.PortPolicy"
+      object_type           = "fabric.PortPolicy"
+      moid                  = intersight_fabric_port_policy.fabric_port_policy1.moid
+      selector              = ""
+    },
+    {
+      additional_properties = ""
```

Comment:
remove all the properties which has empty value

Reply:
Both attributes "additional_properties" and "selector" are required.